logic/scorer/scorer.go

```diff
@@ -23,26 +40,25 @@ func points(lead model.Card, hand []model.Card, isCrib bool) int {
 		return 0
 	}
 
-	values := make([]int, 5)
-	ptValues := make([]int, 5)
-	for i, c := range hand {
-		values[i] = c.Value
-		ptValues[i] = c.PegValue()
+	var (
+		vals   values
+		ptVals values
+	)
+	allCards := [5]model.Card{hand[0], hand[1], hand[2], hand[3], lead}
+	for i, c := range allCards {
+		// building up info for later
+		vals[i] = c.Value
+		ptVals[i] = c.PegValue()
 	}
-	values[4] = lead.Value
-	ptValues[4] = lead.PegValue()
-
-	sort.Ints(values)
-	sort.Ints(ptValues)
```

Comment:
:thinking: how do we get away with not sorting them? :thinking:

Reply:
read on to find out :trollface:

Reply:
I see: it's because `howManyAddUpTo` does a continue instead of a break when `val > target`